livingatlas/configs/la-pipelines.yaml

```diff
@@ -6,15 +6,19 @@ run:
   local:
     # jar: we get the jar from our dev or production environment
     sparkTmp: /data/spark-tmp
+    dwcaTmp: /data/dwca-tmp
+    dwcaImportDir: /data/biocache-load
     sparkMaster: ""
   spark-embedded:
     # jar: we get the jar from our dev or production environment
     sparkTmp: /data/spark-tmp
+    dwcaTmp: /data/dwca-tmp
+    dwcaImportDir: /data/biocache-load
     sparkMaster: ""
   spark-cluster:
     jar: /efs-mount-point/pipelines.jar
     sparkTmp: /data/spark-tmp
-    sparkMaster: spark://aws-spark-quoll-1.ala:7077
+    sparkMaster: spark://localhost:7077
 
```

Comment:
`dwcaImportDir` is empty in `spark-cluster` so I get this error:

```
ls: cannot access '/dr*/dr*.zip': No such file or directory
```
running `./la-pipelines do-all all --dry-run --cluster`

Reply:
thanks @vjrj ill fix this

Reply:
Ive added a default dwcaImportDir and tested. all ok now.